.mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
+ruby = "3.3.4"
```

Comment:
we should remove this file. we use .tools-versions for this.

Reply:
👍